sw/device/tests/dif/meson.build

```diff
@@ -66,6 +66,22 @@ test('dif_i2c_unittest', executable(
   cpp_args: ['-DMOCK_MMIO'],
 ))
 
+test('dif_gpio_test', executable(
+  'dif_gpio_test',
```

Comment:
Sorry, I should have pointed you to update these too.

Reply:
No worries, done.